### ISSUE TITLE
Run docker-compose up

### ISSUE THREAD

**Issue by Headless-Rasputin** [Howchoo/crypto-ticker#3]
Hello,

I am waiting for the hardware to build the LED display and am in the process of configuring the Raspberry pi 3 to use once built.

I receive the following error when on step 9. Run docker-compose up. I had installed docker-composer right before and updated the pi. Not sure what I should do to resolve.  This is all I have in the settings.env file.  I updated settings.env with API=https://coingecko.com/api/documentations/v3/swagger.json
and still get the same error. 

SYMBOL=btc
API=coingecko
REFRESH_RATE=100

 'Error while fetching server API version: {0}'.format(e)
docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', PermissionError(13, 'Permission denied'))

**Comment by josephtyler** [comment]
Can you run `docker ps` without permission errors? If not, did you run through these steps when you installed docker: https://howchoo.com/g/nmrlzmq1ymn/how-to-install-docker-on-your-raspberry-pi#using-docker-as-a-non-root-user?

Let me know.

**Comment by Headless-Rasputin** [comment]
I was able to get this resolved and get the ticker to work! I only have a 16 x 32 LED matrix so the numbers are cut off on my matrix. Is there anyway I can get this to run just on this one LED matrix? 

I edited the frame.py file and updated the led rows to 16 and 32 and changed the ticker.py to only show btc instead of btc and eth. Now my 16x32 matrix is just showing btc across the screen. It would be great if i could get the price to scroll across here too. Not sure how.
